ref(url): update helm's github url

imroc opened the pull request:
https://github.com/kubernetes/helm -> https://github.com/helm/helm
https://github.com/kubernetes/charts -> https://github.com/helm/charts

Signed-off-by: Roc Chan <roc@imroc.io>

bacongobbler reviewed (changes requested):
Can we split out the indentation changes to the `get` script in a separate PR? I'd like to take a look at that one separately but that shouldn't block this from going in. Thanks

imroc committed:
ref(url) update helm's github url

https://github.com/kubernetes/helm -> https://github.com/helm/helm
https://github.com/kubernetes/charts -> https://github.com/helm/charts

Signed-off-by: Roc Chan <roc@imroc.io>

imroc commented:
@bacongobbler  Oops, I didn't notice that the indentation was changed, maybe I was accidentally formatted by the editor when replace all in the whole project. Now I fixed it, PATL.
